packages/tuono/package.json

```diff
@@ -106,6 +106,7 @@
     "@types/node": "^22.0.0",
     "@vitejs/plugin-react-swc": "^3.7.0",
     "fast-text-encoding": "^1.0.6",
+    "message-port-polyfill": "^0.2.0",
```

Comment:
Is there any better library? Seems pretty unmaintained https://github.com/rocwind/message-port-polyfill

Reply:
We could consider to creare our pollyfill since this is something we have to rely on to support react@19

Reply:
our pollyfill = fork that library (in here on in another repo) and use it here 😅

Reply:
yeah, it seems the library is simple enough that we can just fork it as a single file polyfill. we can create a `polyfills` folder in `packages/tuono/bin` for future polyfills files that we want to maintain ourselves

Reply:
How about exporting the polyfill from `tuono/ssr`?

```diff
- MessageChannel: ['message-port-polyfill', 'MessageChannelPolyfill'], 
+ MessageChannel: ['tuono/ssr', 'MessageChannelPolyfill'],
```

I would add the polypill in a new `packages/tuono/src/ssr/polifylls` folder.
If you can add also some unit test it would be awesome!

@mustafasegf Would you like to try implementing this change?

Reply:
sure i can do that